### DISCUSSION TITLE
Only 2 decimals displayed in legend when moving the mouse.

### DISCUSSION THREAD

**Discussion by heydude070** [highfestiva/finplot#252] [Q&A] [answered]
First of ALL THANKS, For awesome plotting library 🥇  

Only 2 decimals when updating crosshair and legend when moving the mouse, While Currency Pairs quoted in 4 to 5 decimals. 

Pic : https://pasteboard.co/KjypE3L.png
Code : https://pastebin.pl/view/d8081b87

Can you make to 5 decimals values for OCHL to display in legend.

Thanks. 👍

**Comment by highfestiva** [comment, marked as answer]
Replace `%%.2f` with `%%.5f`?

**Reply by heydude070** [reply]
Hi,
Is It Possible paste any kind of visualization with different color on Chart/Candlestick based on some condition e,g Crossover.
Can I change the background color of the plot based on some condition. e,g Crossover.
How can I add indicator values on legend, below OHLC valves. 

You can use the above code.

Thanks.
